flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/DispatcherResourceCleanupTest.java

```diff
@@ -579,10 +587,15 @@ public void testErrorHandlingIfJobCannotBeMarkedAsCleanInJobResultStore() throws
         final CompletableFuture<JobResultEntry> dirtyJobFuture = new CompletableFuture<>();
         final JobResultStore jobResultStore =
                 TestingJobResultStore.builder()
-                        .withCreateDirtyResultConsumer(dirtyJobFuture::complete)
+                        .withCreateDirtyResultConsumer(
+                                jobResultEntry -> {
+                                    dirtyJobFuture.complete(jobResultEntry);
+                                    return FutureUtils.completedVoidFuture();
+                                })
                         .withMarkResultAsCleanConsumer(
                                 jobId -> {
-                                    throw new IOException("Expected IOException.");
+                                    return FutureUtils.completedExceptionally(
```

Comment:
This can be shortened (the brackets are obsolete in a oneliner). Analogously to your change above.

Reply:
Fixed.